projects/canopy/src/lib/paginator/paginator.component.scss

```diff
@@ -0,0 +1,35 @@
+.lg-paginator {
+  align-items: center;
+  display: flex;
+  justify-content: flex-end;
+
+  &__page-size {
+    align-items: center;
+    display: flex !important;
+    margin: 0 var(--space-xs) 0 0 !important;
```

Comment:
We try to avoid `!important`, was there a reason why you added it? can it be solved with specificity?

Reply:
I'll have to have a look, I think most of this CSS will be junk once there is a design for the component so I probably just bodged it in